src/test/clojure/clara/test_rules.clj

```diff
@@ -1330,7 +1330,23 @@
         (insert [1 :foo 42])
         (fire-rules))
 
-    (is (= 42 @rule-output))))
+    (is (= 42 @rule-output-env))))
+
+(deftest test-destructured-test-env-binding
+  (let [rule-output-env (atom nil)
+        rule {:name "clara.test-destructured-binding/test-destructured-test-env-binding"
+              :env {:rule-output rule-output-env} ; Rule environment so we can check its output.
+              :lhs '[{:args [[e a v]]
+                      :type :foo
+                      :constraints [(= e ?entity) (= v ?value)]}
+                     {:constraints [(= ?entity 1) (reset! rule-output ?value)]}]
+              :rhs '(println @rule-output)}]
```

Comment:
I think we can just have a RHS here that doesn't really do anything.   For example an operation like
```(inc 1)``` or anything similar.  In order to resemble real rules it is probably better to have something to execute though rather than say ```(do nil)``` although that would work too.

 This adds some clutter to the logs of the test build output.

From https://github.com/cerner/clara-rules/actions/runs/5152272254/jobs/9278212444 :
````
lein test clara.test-rules
42
42
````

Reply:
Makes sense, will make this change.